### PR TITLE
wip prep

### PR DESCRIPTION
**Pull request by jshawl**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -51,52 +45,37 @@ As a result, this allows changes to the browser's URL to drive navigation throug
...
-
-Notice there's an `app.js` linked in this `index.html`.  Let's open it and add this line:
+#### Inject the `ui.router` dependency
 
 ```js
-angular.module("grumblr", ["ui.router"]);
+angular.module("stoplight", ["ui.router"])
 ```
 
 Here we've done two things: we've created a new module called `grumblr`, and we've told it to require another module called `ui.router`. What we see in the array brackets is Angular's way of "requiring" modules.
```
Comment (amaseda) [inline readme.md]: Still mentions grumblr

File: readme.md [hunk 2]
```diff
@@ -51,52 +45,37 @@ As a result, this allows changes to the browser's URL to drive navigation throug
 
 ## Let's Build an Angular App
 
-Today, we are going to build off of what we learned in the intro class, and construct an app from scratch utilizing `uiRouter`. We're going to look at how we can build a SPA with multiple views and controllers.
-
-## Getting set up (5 / 20)
+Today, we are going to build off of what we learned in the intro class, and represent state utilizing `uiRouter`.
```
Comment (nolds9) [inline readme.md:48]: could be an opportunity to define what we mean by state..or a quick example

File: lab.md [hunk 3]
```diff
@@ -0,0 +1,498 @@
...
+Now if you go to `localhost:8080` you should see your app.
+
+#### Open the file in Atom and your browser
+
+It's all terribly exciting: the word `Grumblr`, and that's pretty much it.
+
+Grumblr is like Tumblr, only grumblier. Eventually, it will be a two-model CRUD app with posts and comments.
+
+In the coming classes you're going to be interacting with data from an API that we provide. For this class, though, we'll just be hard-coding data and getting some views up and running.
+
```
Comment (nolds9) [inline lab.md]: Think we can remove this section about upcoming classes

File: lab.md [hunk 4]
```diff
@@ -0,0 +1,498 @@
...
+</section>
+
+## Create a module for grumblr and inject `ui.router`
+
+<section markdown="1">
+
+```js
+// app.js
+angular.module("grumblr", ["ui.router"]);
+```
```
Comment (nolds9) [inline lab.md:76]: these syntax highlightings don't seem to show up fyi

File: lab.md [hunk 5]
```diff
@@ -0,0 +1,498 @@
...
+</section>
+
+## Loop through data and display info about each grumble
+
+<section markdown="1">
+
+```html
+<!-- js/grumbles/index.html -->
+<div data-ng-repeat="grumble in GrumbleIndexViewModel.grumbles">
+  <p>{{grumble.title}}</p>
```
Comment (nolds9) [inline lab.md]: Weird but I think these braces were not escaped...the entire expression doesn't show up on the rendered html

File: lab.md [hunk 6]
```diff
@@ -0,0 +1,498 @@
...
+```
+</section>
+
+## Show template
+
+<section markdown="1">
+
+```html
+<h2>{{GrumbleShowViewModel.grumble.title}}</h2>
+```
```
Comment (nolds9) [inline lab.md:423]: same problem as above comment, expression not rendered in html view

**Pull request merged**